docs/templates/getting-started/minimizing_functions.md

```diff
@@ -181,4 +214,4 @@ It's normal if this doesn't make a lot of sense to you after this short tutorial
 but I wanted to give some mention of what's possible with the current code base,
 and provide some terms to grep for in the hyperopt source, the unit test,
 and example projects, such as [hyperopt-convnet](https://github.com/jaberg/hyperopt-convnet).
```

Comment:
I know this is not from your PR, but would you please replace all `jaberg` references by `hyperopt`? thanks!

Reply:
Good idea, they've been replaced now